ui/src/containers/EquipmentContainer.jsx

```diff
@@ -46,142 +46,136 @@ import HarvesterMaintenance from '../components/Equipment/HarvesterMaintenance';
-                      unload={this.props.unmountSample}
-                      abort={this.props.abort}
-                      scan={this.props.scan}
-                      contents={this.props.contents}
-                      refresh={this.props.refresh}
-                    />
-                  </Col>
-                  <Col sm={6}>
-                    <SampleChangerMaintenance
-                      commands={this.props.commands}
-                      global_state={this.props.global_state}
-                      commands_state={this.props.commands_state}
-                      message={this.props.message}
-                      sendCommand={this.props.sendCommand}
-                    />
-                  </Col>
-                </Row>
-              )}
+              <Row className="row">
+                <Col sm={9}>
+                  <Harvester
+                    state={props.haState}
+                    harvestCrystal={props.harvestCrystal}
+                    harvestAndLoadCrystal={props.harvestAndLoadCrystal}
+                    abort={props.haAbort}
+                    contents={props.haContents}
+                    handleRefresh={props.haRefresh}
+                  />
+                </Col>
+                <Col sm={3}>
+                  <HarvesterMaintenance
+                    contents={props.haContents}
+                    commands={props.haCommands}
+                    global_state={props.haGlobal_state}
+                    commands_state={props.haCommands_state}
+                    message={props.haMessage}
+                    sendCommand={props.haSendCommand}
+                    calibratePin={props.haCalibratePin}
+                    sendDataCollectionToCrims={props.sendDataCollectionToCrims}
+                    validateCalibration={props.haValidateCalibration}
+                  />
+                </Col>
+              </Row>
             </GenericEquipment>
-            {this.props.haContents.use_harvester ? (
-              <GenericEquipment
-                state={this.props.haState}
-                name={this.props.haContents?.name}
-                CollapseOpen
-              >
-                <Row className="row">
-                  <Col sm={9}>
-                    <Harvester
-                      state={this.props.haState}
-                      harvestCrystal={this.props.harvestCrystal}
-                      harvestAndLoadCrystal={this.props.harvestAndLoadCrystal}
-                      abort={this.props.haAbort}
-                      contents={this.props.haContents}
-                      handleRefresh={this.props.haRefresh}
-                    />
-                  </Col>
-                  <Col sm={3}>
-                    <HarvesterMaintenance
-                      contents={this.props.haContents}
-                      commands={this.props.haCommands}
-                      global_state={this.props.haGlobal_state}
-                      commands_state={this.props.commands_state}
```

Comment:
Bug here: `commands_state` refers to `state.sampleChangerMaintenance.commands_state`. I think it's supposed to be `haCommands_state` which refers to `state.harvesterMaintenance.commands_state`.

Reply:
Probably yes, well spotted it seems very likely indeed. Maybe @jbflo can comment when he is back from vacation if its not the case.